docker/files/srcline.c

```diff
@@ -0,0 +1,920 @@
+// SPDX-License-Identifier: GPL-2.0
```

Comment:
@glpuga the GPL is pervasive, quoting:

> 2. You may modify your copy or copies of the Program or any portion of it [...] provided that you also meet all of these conditions:
> b) You must cause any work that you distribute or publish, that in whole or **in part contains** or is derived from the Program or any part thereof, to be licensed as a whole at no charge to all third parties under the terms of this License.

There _is_ a notion of separate work in GPL terms and conditions, but I'm no lawyer, and this smells as a liability. How about we relocate this to a different repository? A minimal Github Actions workflow to push to Dockerhub should do to make it available to Beluga.

Reply:
Hmm... good catch. I flew totally past me checking the license implications of adding the file replacement.

This is not essential, so I'll take down this PR while we figure if it's worth adding it back in some other form.